web/public/templates/dojos/events/user-events.dust

```diff
@@ -24,7 +24,7 @@
             <tr ng-click="showEventInfo(dojoEvents, $index)" class="event-info-row">
               <td>{{ event.name }}</td>
               <td>{{ event.formattedDate }}</td>
-              <td>{{ event.type }}</td>
+              <td>{{ getEventTypeDisplayText(event) }}</td>
```

Comment:
Double binding on a function returning a translation, largely overkill, prefer storing it in a var and binding once